src/scripts/browserDeviceProfile.js

```diff
@@ -918,22 +918,22 @@ export default function (options) {
     let vp9VideoRangeTypes = 'SDR';
     let av1VideoRangeTypes = 'SDR';
 
+    if (supportsDolbyVision(options)) {
+        hevcVideoRangeTypes += '|DOVI';
+    }
+
```

Comment:
I prefer not to make unnecessary changes. Please move it back to its original location (your regex won't break).